### DISCUSSION TITLE
How to change default Laragon path for some components

### DISCUSSION THREAD

**Discussion by e4rthdog** [leokhoa/laragon#904] [General]
I want to use another python installation and not the one included in Laragon. Is there a way to mopdify the default path and remove the python entries so i can use Laragon system wide without the need of its own cmdr?

![image](https://github.com/user-attachments/assets/2f85da9b-7107-4471-9bed-6b7963791b11)

**Comment by Pen-y-Fan** [comment]
I installed new python versions in Laragon/bin/python directory and then let Laragon switch versions.

If you want to be completely independent, remove the python directory from laragon/bin and install python to another/default directory. the python installer should register the path of that installation.

You may need to remove laragon from path, then delete the python directory and add laragon to path again, should you want to use Laragon with another terminal. See [laragon.org/docs/isolated](https://laragon.org/docs/isolated) for more details.

**Reply by e4rthdog** [reply]
> the python installer should register the path of that installation.

Did not know that! Clever... It seems that whatever Laragon doesn't find in the bin directory is not added to the path. So i can keep Node, Python and Git in my own installations and keep mysql, php and the rest under Laragon.
